geode-serialization/src/test/java/org/apache/geode/internal/serialization/KnownVersionJUnitTest.java

```diff
@@ -44,18 +39,21 @@ public void testKnownVersionClass() throws Exception {
     compare(KnownVersion.GEODE_1_10_0, KnownVersion.GEODE_1_9_0);
     compare(KnownVersion.GEODE_1_11_0, KnownVersion.GEODE_1_10_0);
     compare(KnownVersion.GEODE_1_12_0, KnownVersion.GEODE_1_11_0);
-    compare(KnownVersion.GEODE_1_13_0, KnownVersion.GEODE_1_12_0);
-    compare(KnownVersion.GEODE_1_14_0, KnownVersion.GEODE_1_13_0);
+    compare(KnownVersion.GEODE_1_12_1, KnownVersion.GEODE_1_12_0);
+    compare(KnownVersion.GEODE_1_13_0, KnownVersion.GEODE_1_12_1);
+    compare(KnownVersion.GEODE_1_13_1, KnownVersion.GEODE_1_13_0);
+    compare(KnownVersion.GEODE_1_14_0, KnownVersion.GEODE_1_13_1);
   }
 
   private void compare(KnownVersion later, KnownVersion earlier) {
     assertTrue(later.compareTo(earlier) > 0);
-    assertTrue(later.equals(later));
-    assertTrue(later.compareTo(later) == 0);
+    assertEquals(later, later);
+    // noinspection EqualsWithItself for testing
+    assertEquals(0, later.compareTo(later));
     assertTrue(earlier.compareTo(later) < 0);
 
     assertTrue(later.compareTo(Versioning.getVersion(earlier.ordinal())) > 0);
-    assertTrue(later.compareTo(Versioning.getVersion(later.ordinal())) == 0);
+    assertEquals(0, later.compareTo(Versioning.getVersion(later.ordinal())));
     assertTrue(earlier.compareTo(Versioning.getVersion(later.ordinal())) < 0);
```

Comment:
Instead of `assertTrue()`, might it be better to use `assertThat().isGreaterThan()` and `assertThat().isLessThan()` for these checks?

Reply:
Static analyzer didn't pick that one up... Thanks!